GitHub/1/README.md

```diff
@@ -7,12 +7,11 @@
 ## Challenge
 - Fork devopsloft/training repo
 - Clone it
-- Checkout to a branch
-- Implement a meaningful change to this challenge
+- Pick up an issue and implement it
```

Comment:
@lioramilbaum I think we should create a feature branch.
It's never too early to adopt best practices  :)

Reply:
ack